app/scripts/modules/core/src/pipeline/config/parameters/Parameter.tsx

```diff
@@ -107,113 +121,124 @@ export class Parameter extends React.Component<IParameterProps> {
       <div className="parameter-config">
         <div className="row">
           <div className="col-md-12">
-            <div className="form-horizontal panel-pipeline-phase">
-              <div className="row name-field">
-                <div className="col-md-1">{isMultiple && <DragHandler />}</div>
-                <label className="col-md-2 sm-label-right">
-                  <span className="label-text">Name</span>
-                </label>
-                <div className="col-md-9">
-                  <div className="row">
-                    <div className="col-md-4">
-                      <input
-                        className="form-control input-sm"
-                        type="text"
-                        required={true}
-                        value={name}
-                        onChange={handleNameChange}
-                      />
-                    </div>
-                    <div className="col-md-2 sm-label-right">
-                      <span className="label-text">Label </span>
-                      <HelpField id="pipeline.config.parameter.label" />
-                    </div>
-                    <div className="col-md-4">
-                      <input className="form-control input-sm" type="text" value={label} onChange={handleLabelChange} />
-                    </div>
-                    <div className="col-md-1 col-md-offset-1">
-                      <Tooltip value="Remove parameter">
-                        <button className="btn btn-link glyphicon glyphicon-trash" onClick={removeParameter} />
-                      </Tooltip>
-                    </div>
-                  </div>
-                </div>
-              </div>
-
-              <StageConfigField label="Required">
-                <div className="checkbox">
-                  <label>
-                    <input type="checkbox" checked={required} onChange={handleRequiredChange} />
+            <fieldset disabled={inherited} className={classNames({ 'templated-pipeline-item': inherited })}>
```

Comment:
this large diff comes from indenting pre-existing elements in the added `<fieldset>` container and from making the delete button conditional to inheritance (line 154)